ghost/email-events/lib/EmailBouncedEvent.js

```diff
@@ -1,7 +1,3 @@
-/**
- * @typedef {import('bson-objectid').default} ObjectID
- */
-
 module.exports = class EmailBouncedEvent {
```

Comment:
I've realized that these events don't confirm to the `IEvent` interface defined by `@tryghost/domain-events` which is my :b:

I think they can go in as-is, and we can either rework the events, or rework the `IEvent` interface to be more generic during cleanup - I've added a TODO to come back to this next week either way ☺️